### PR TITLE
RF | guidePageSpec > AT_008.002 | Main menu > Guide | Verify the first button "Learn more" is clickable and user will be redirected new url

### PR DESCRIPTION
**Pull request by SvetlanaG1011**
https://trello.com/c/PRJSYaHQ/820-rf-guidepagespec-at008002-main-menu-guide-verify-the-first-button-learn-more-is-clickable-and-user-will-be-redirected-new-url

### CODE REVIEW

**Review by alena-resh — DISMISSED** [review]

File: cypress/fixtures/url.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "facebookUrl": "270748973021342",
     "mapsPage": "https://openweathermap.org/weathermap",
     "mainAbout": "https://openweather.co.uk/#main_about",
+    "apiCurrent": "https://openweathermap.org/api#current",
```
Comment (TimothyHuxx) [inline cypress/fixtures/url.json:23]: Please move your code to the end.

Comment (SvetlanaG1011) [reply cypress/fixtures/url.json]: moved. thank you.

Comment (TimothyHuxx) [reply cypress/fixtures/url.json]: @SvetlanaG1011 don't forget to push your changes. I still don't see the update

**Review by TimothyHuxx — APPROVED** [review]
Approved

File: cypress/fixtures/url.json (hunk 1 above)
Comment (TimothyHuxx) [reply cypress/fixtures/url.json]: Good job

**Review by shabaiev — APPROVED** [review]
krasavchik 🤌🏻

**Pull request merged**